app/Services/Logging/DebugRequestLogger.php

```diff
@@ -0,0 +1,43 @@
+<?php
+
+namespace App\Services\Logging;
+
+use Illuminate\Http\Request;
+
+class DebugRequestLogger extends AbstractRequestLogger
+{
+    protected function prepareMessage(): string
+    {
+
+        return 'Useful information about Request';
+    }
+
+    protected function extractRequestData(Request $request): array
+    {
+        $url = $request->url();
+        $urlWithQueryString = $request->fullUrl();
+        $uri = $request->path();
+        $input = $request->all();
+        $method = $request->method();
+        $value = $request->header('X-Header-Name');
+        $ipAddress = $request->ip();
+        $token = $request->bearerToken();
```

Comment:
Let's discard token, acceptableContentTypes and header, ads it doesn't provide to much value.

Reply:
done in the 3rd commit